### ISSUE TITLE
[uploaded.net] : "Free Download" option not working with plug-in enabled

### ISSUE THREAD

**Issue by Ano11** [EFForg/https-everywhere#9847]

**Comment by Bisaloo** [comment]
Thank you for your report!

Could you provide an URL where we can see this behaviour please?

**Comment by Ano11** [comment]
Hi there,

 

For example I visit this page: http://uploaded.net/file/kdbnz4bu

Then there is a button "Free Download".

I press it and have to wait for 30 seconds.

 

With the "https-everywhere plugin" enabled, after waiting for 30 seconds this message appears: "The free download is currently not available - Please try again later!" and I cannot download the file.

 

When I disable the plugin, after 30 seconds a Captcha appears and after solving it I can download the file without any problem.

 

Thanks!

**Comment by jeremyn** [comment]
(I edited the previous comment to remove some email stuff.)

**Comment by Bisaloo** [comment]
The problem is partially fixed (as in you get the captcha but nothing happens next) by adding the following exclusion rule:

```xml
     <exclusion pattern="http://uploaded.net/io/ticket/slot/*" />
```

but given that the website also redirects to `http` on major parts of the website, I am wondering whether it may not be better to completely `default_off` this ruleset...

**Comment by Bisaloo** [comment]
Update: this issue is fully solved by adding

```xml
     <exclusion pattern="http://uploaded.net/io/ticket/*" />
```

If someone wants to optimize this regex, in the case of http://uploaded.net/file/kdbnz4bu, we want to load

- http://uploaded.net/io/ticket/slot/kdbnz4bu
- http://uploaded.net/io/ticket/captcha/kdbnz4bu

over `http`.

**Comment by J0WI** [comment]
closed by #9936

**Comment by Ano11** [comment]
Great, thanks!
